Floofbot/Services/EventLoggerService.cs

```diff
@@ -595,5 +602,37 @@ public Task UserUnmuted(IUser user, IUser unmuter, IGuild guild)
             });
             return Task.CompletedTask;
         }
+
+        class WordFilterService
+        {
+            List<FilteredWord> _wordfilterList;
+
+            bool hasFilteredWord(FloofDataContext floofDb, string messageContent, ulong serverId, ulong channelId)
+            {
+                // whitelist means we don't have the filter on for this channel
+                if (floofDb.FilterChannelWhitelists.AsQueryable()
+                    .Any(x => x.ChannelId == channelId && x.ServerId == serverId))
+                {
+                    return false;
+                }
+
+                if (_wordfilterList == null)
+                {
+                    _wordfilterList = floofDb.FilteredWords.AsQueryable()
```

Comment:
only issue I see is if we add a new filtered word we have to restart bot?

Reply:
oh true. We don't have some time-to-live or reload filter cache command here yet